app/controllers/instructeurs/procedures_controller.rb

```diff
@@ -58,17 +58,14 @@ def show
         .without_followers
         .en_cours
 
-      @followed_dossiers = current_instructeur
+      followed_dossiers = current_instructeur
         .followed_dossiers
         .where(groupe_instructeur: current_instructeur.groupe_instructeurs)
         .for_procedure(procedure)
         .en_cours
 
-      @followed_dossiers_id = current_instructeur
-        .followed_dossiers
-        .where(groupe_instructeur: current_instructeur.groupe_instructeurs)
-        .for_procedure(procedure)
-        .pluck(:id)
+      @followed_dossiers_count = followed_dossiers.count
```

Comment:
En le mettant apres la ligne 68

```suggestion
      @followed_dossiers_count = followed_dossiers_id.count
```